### PR TITLE
Implemented fft in paddle

### LINKED ISSUE #16684: fft

### PR DESCRIPTION
**Pull request by KevinUli**
Close #16684

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
 # local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="paddle.fft.fft",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        shape=(2,),
```
Comment (xoiga123) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py:13]: Please widen the shape range (their example is `(7,)`) :1234:

**Review by xoiga123 — CHANGES_REQUESTED** [review]
Maybe make the test more generalized a bit :muscle:

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py (hunk 1 above)
Comment (KevinUli) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py]: I've widened the range

Comment (xoiga123) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py]: Does this work for any `int` from 2 to 7? Maybe we should do 
`shape=st.sampled_from(None | st.integers)`

Comment (KevinUli) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_paddle_fft.py]: I've removed `shape` and added `None` for `n`

**Review by xoiga123 — APPROVED** [review]
Thanks @KevinUli :clap:

**Pull request merged**